src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -44,6 +44,11 @@ public final class ElevatorConstants {
     public static final Mass carriageMass = Pounds.of(20);
     public static final Distance drumRadius = Inches.of(1.503);
 
```

Comment:
Just as a heads up, this is the drum diameter,  not the drum radius.

Reply:
That could easily be a contributing factor haha! Thanks for letting me know